.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: python
+python:
```

Comment:
We should make sure this matches what was done in asciigraf for python 3.7

Reply:
I updated the jobs to include Python 3.6 and Python 3.7